webpack.config.js

```diff
@@ -3,9 +3,10 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 module.exports = {
     entry: {
         customfunctions: ["./src/customfunctions.js"],
+        customfunctionsjson: ["./config/customfunctions.json"],
```

Comment:
I don't think there should be a comma after this line. (I could be completely wrong about that though...)